chromadb/db/clickhouse.py

```diff
@@ -18,7 +18,7 @@
     {"embedding": "Array(Float64)"},
     {"document": "Nullable(String)"},
     {"id": "Nullable(String)"},
-    {"metadata": "String"},
+    {"metadata": "Nullable(String)"},
```

Comment:
I'm not sure if this is a backwards compatible change. I.E code now assumes strings will be None whereas before they were the empty string?

Reply:
Discussed offline; this implies a change in logic (checking empty string VS checking for `None`), but should be fine in practice.